lib/strong_migrations/version.rb

```diff
@@ -1,3 +1,3 @@
 module StrongMigrations
-  VERSION = "0.4.0"
+  VERSION = "0.4.1"
```

Comment:
It's best to leave versioning to maintainers.

Reply:
Old habits...